apps/frontend/src/app/(app)/space/this-week/page.tsx

```diff
@@ -1,5 +1,137 @@
+"use client"
+
+import { Icon } from "@iconify-icon/react"
+
+import React from "react"
+import { LeftChevron, RightChevron } from "@/src/lib/icons/Navigation"
+
 const ThisWeekPage: React.FC = () => {
-  return <div className="text-gray-color p-16">this week page</div>
+  return (
```

Comment:
Render just one Main component here rather than the entire logic Lik
`<div className="size-full overflow-auto bg-background p-16 text-secondary-foreground">
<ThisWeekComponent/>
</div>
`

And then create separate components inside ThisWeekComponent if required

Reply:
nice, thanks